#5 new atmosphere model

Aarush26 opened the pull request:
Created a function that calculates the temperature, pressure, and density of the atmosphere based on the input of a given altitude to address issue #5. The atmospheric model function is based off the NASA model [here](https://www.grc.nasa.gov/WWW/K-12/airplane/atmosmet.html). We are only concerned with the troposphere equations and constants, since we probably will not fly above 11,000 meters (we can dream though).

Commit:
Create AtmosphereModel.m

INITIAL COMMIT: Added atmosphere model based on NASA model

Commit:
Update AtmosphereModel.m

Updated explanation of what the code means and represents in relation to the atmosphere model.

TheGreatFez commented on Simulation/AtmosphereModel.m:
```diff
@@ -0,0 +1,28 @@
+% Atmosphere model is sourced from NASA at this link:
+% https://www.grc.nasa.gov/WWW/K-12/airplane/atmosmet.html
+
+% Create input altitude
```
Let's remove this comment since we changed the code to a function

TheGreatFez reviewed (changes requested):
Just one small change

Commit:
Removed comment

Took out the comment above the altitude function since it was unrelated to the code below it

Aarush26 reviewed (commented):
Deleted the comment that was above the function code because it referred to part of the code that was changed to make a function that used altitude as an input.

TheGreatFez reviewed (approved):
Looks great!
